nixos/modules/config/users-groups.nix

```diff
@@ -473,14 +475,15 @@ in {
       '';
     };
 
-    # FIXME: obsolete - will remove.
-    security.initialRootPassword = mkOption {
-      type = types.str;
-      default = "!";
-      example = "";
-      visible = false;
-    };
 
+    users.allowNoPasswordLogin = mkOption {
+      type = types.bool;
+      default = false;
+      description = ''
+        Do not require that at least the `root` user or a user in the `wheel` group can log in using 
+        a password or an SSH key.
+      '';
+    };  
```

Comment:
Neither name nor description make it clear that this is about disabling an assertion.
How about `users.disableLoginPossibilityAssertion`?

The assertion message can mention this option, to improve discoverability.

Reply:
Not the best name. Continued in sibling thread https://github.com/NixOS/nixpkgs/pull/73106#discussion_r685182949. Resolving this thread.